### PR TITLE
Add account switcher in search bar

### PR DESCRIPTION
**Pull request by AndyScherzinger**
light|dark
---|---
light|dark

Resolves parts of #909 and kind of #1058

**TODO**
* [x] cleanup UI (move strings to `strings.xml`, same for dimens and add content descriptions for a11y)
* [x] fix spotbugs
* [x] remove duplicate settings from settings screen that are now available in account swicther
* [x] respect single-account-flag

Signed-off-by: Andy Scherzinger <info@andy-scherzinger.de>

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Very very very very very nice! :D

**Comment by nickvergessen** [comment]
✨

File: app/build.gradle [hunk 1]
```diff
@@ -150,6 +150,10 @@ android {
         htmlOutput file("$project.buildDir/reports/lint/lint.html")
         disable 'MissingTranslation'
     }
+
+    buildFeatures {
+        viewBinding true
```
Comment (mahibi) [inline app/build.gradle:155]: 🚀

**Review by mahibi — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (AndyScherzinger) [reply app/build.gradle:155]: ❤️ Yeah, forgot to mention that I added it for this PR/dialog and will continue to move non-conductor related views to migrate to this. Conductor-related classes will probably migrated to the new 3.0 concepts in one step :) (just a guess though)

**Pull request merged**